assets/onboarding/features/features-selection.jsx

```diff
@@ -50,17 +55,20 @@ const FeaturesSelection = ( {
 						}
 						onChange={ toggleItem( slug ) }
 						checked={ selectedSlugs.includes( slug ) }
-						disabled={ status === INSTALLED_STATUS }
-						className={ `sensei-onboarding__checkbox ${
-							status === INSTALLED_STATUS
-								? 'installed-status'
-								: ''
-						}` }
+						disabled={ !! status }
+						className={ classnames( 'sensei-onboarding__checkbox', {
+							'installed-status': status === INSTALLED_STATUS,
+						} ) }
```

Comment:
Another approach is adding a class like `` `status-${status}` ``, to have classes for all statuses with less code.

Reply:
Good idea! https://github.com/Automattic/sensei/pull/3190/commits/144c4939020e76fd63589d9b9849d0e123ae35ab